velox/dwio/dwrf/reader/StripeStream.h

```diff
@@ -91,6 +91,10 @@ class StripeStreams {
  public:
   virtual ~StripeStreams() = default;
 
+  virtual FileFormat getFormat() const {
```

Comment:
why not leaving it as abstract?

Reply:
Yes it should be abstract.
Moved this to `StripeStreamsBase`.